src/components/CheckboxGroup/README.md

```diff
@@ -28,13 +28,13 @@ In order to override the theme, use Wix Stylable extends capabilities and wrap w
     ``` javascript
     import CheckboxGroup from 'wix-ui-tpa/CheckboxGroup';
     import Checkbox from 'wix-ui-tpa/Checkbox';
-    import checkboxGroupStylesExt from './CheckboxGroupExt.st.css';
+    import { st, classes } from './CheckboxGroupExt.st.css';
 
     const CheckboxExt = (props) => (
         <CheckboxGroup
             label="Label"
+            className={st(classes.root)} 
             {...props}
```

Comment:
```suggestion
            {...props}
            className={st(classes.root, props.className)}
```

Reply:
again, an example of extending a component